api/iampap/iampapv1/iam_policy.go

```diff
@@ -0,0 +1,102 @@
+package iampapv1
+
+import (
+	"fmt"
+	"github.com/IBM-Bluemix/bluemix-go/client"
+)
+
+type AccessPolicyRequest struct {
+	Roles     []Roles     `json:"roles" binding:"required"`
+	Resources []Resources `json:"resources" binding:"required"`
+}
+
+type AccessPolicyResponse struct {
+	ID        string
+	Roles     []Roles
+	Resources []Resources
+}
+
+type AccessPolicyListResponse struct {
+	Policies []AccessPolicyResponse
+}
+
+type Roles struct {
+	ID          string `json:"id" binding:"required"`
+	DisplayName string `json:"displayName,omitempty"`
+	Description string `json:"description,omitempty"`
+}
+
+type Resources struct {
+	ServiceName     string `json:"serviceName,omitempty"`
+	ServiceInstance string `json:"serviceInstance,omitempty"`
+	Region          string `json:"region,omitempty"`
+	ResourceType    string `json:"resourceType,omitempty"`
+	Resource        string `json:"resource,omitempty"`
+	SpaceId         string `json:"spaceId,omitempty"`
+	AccountId       string `json:"accountId,omitempty"`
+	OrganizationId  string `json:"organizationId,omitempty"`
+}
+
+type IAMPolicy interface {
+	Create(scope, userId string, params AccessPolicyRequest) (AccessPolicyResponse, string, error)
+	List(scope, userId string) (AccessPolicyListResponse, error)
+	Delete(scope, userId, policyId string) error
+	Find(scope, userId, policyId string) (AccessPolicyResponse, error)
```

Comment:
I think it should be called `Get` since you have userId and policyId and you are retrieving it value